runtimes/ocaml/runtime.mli

```diff
@@ -29,7 +29,11 @@ type integer = Z.t
 type decimal = Q.t
 type date = Dates_calc.Dates.date
 type duration = Dates_calc.Dates.period
-type date_rounding = Dates_calc.Dates.date_rounding
+
+type date_rounding = Dates_calc.Dates.date_rounding =
+  | RoundUp
```

Comment:
Why do you need to do that? To reexport the constructor names?

Reply:
Yes, the generated code uses `RoundUp` directly, and the constructor wasn't in scope ; the other fix would have been to replace that with `Dates_calc.Dates.RoundUp` but this seemed nicer.

Reply:
I wasn't even aware of the possibility of doing that in OCaml ! It's indeed nicer :)